src/main/java/me/coley/recaf/parse/bytecode/ast/NameAST.java

```diff
@@ -27,6 +29,13 @@ public NameAST(int line, int start, String name) {
 	public String getName() {
 		return name;
 	}
+	
+	/**
+	 * @return Name without escapes.
+	 */
+	public String getUnescapedName() {
+		return EscapeUtil.unescape(name);
+	}
```

Comment:
Something about having two name getters feels wrong. I get that this works, but its just a gut feeling.

Reply:
I wanted to maintain compatibility with other parts of the code that may use names without escaping